app/main.py

```diff
@@ -1,8 +1,36 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.wife = None
+        self.husband = None
+        self.__class__.people[name] = self
```

Comment:
```suggestion
        self.people[name] = self
```
you also can use self for access class attributes and it is more clear